### PR TITLE
Integration test for Create file

### PR DESCRIPTION
**Pull request by abhinav2603**
Integration test for Create file

### CODE REVIEW

File: tools/integration_tests/implicitdir/file_test.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+// Provides integration tests for write flows with implicit_dir flag set.
+package implicitdir_test
+
+import (
+	"os"
+	"testing"
+)
+
+func TestCreateFile(t *testing.T) {
```
Comment (vadlakondaswetha) [inline tools/integration_tests/implicitdir/file_test.go:23]: createFile is a write flow only. lets add this to write_test.go file. Also please check the contents of the file.

Comment (abhinav2603) [reply tools/integration_tests/implicitdir/file_test.go:23]: So, will all the other tests (renaming, copying, file stat) be also under write_test?

Comment (vadlakondaswetha) [reply tools/integration_tests/implicitdir/file_test.go:23]: nope. lets put them under file_operations_test.go

File: tools/integration_tests/implicitdir/write_test.go [hunk 2]
```diff
@@ -67,3 +67,15 @@ func TestWriteAtRandom(t *testing.T) {
...
+func TestCreateFile(t *testing.T) {
+	fileName := createTempFile()
+
+	// Stat the file to check if it exists.
+	if _, err := os.Stat(fileName); err != nil {
+		t.Errorf("File not found, %v", err)
+	}
+
+	compareFileContents(t, fileName, "line 1\nline 2\n")
+
```
Comment (vadlakondaswetha) [inline tools/integration_tests/implicitdir/write_test.go:80]: nit: remove new line

**Review by vadlakondaswetha — APPROVED** [review]

**Review by vadlakondaswetha — APPROVED** [review]

**Pull request merged**